src/test/isolation2/expected/lockmodes.out

```diff
@@ -2186,6 +2186,14 @@ ROLLBACK
 -- when running concurrently with index creation, for Heap tables.
 -- For AO/CO tables, refer to create_index_allows_readonly.source.
 
+-- TODO-CHERRYPICK: uncomment this, when https://github.com/greenplum-db/gpdb-archive/commit/3be17112144 cherry-picked
```

Comment:
use the `CBDB_*_MERGE_FIXME` replace `TODO-CHERRYPICK` pls.

Reply:
Sure, will do

Reply:
Done.